src/scripts/settings/config.js

```diff
@@ -0,0 +1,159 @@
+(function () {
```

Comment:
If it is feature related maybe feature-config.js. I'm thinking new devs could be confused by settings.js and config.js. They are sort of the same thing from the file name. Maybe a comment in config.js explaining what it is for would be fine.